### DISCUSSION TITLE
Feature Request: Add Field Type 'Database Field' and Sync option

### DISCUSSION THREAD

**Discussion by coercoer** [pods-framework/pods#6710] [Ideas]
## Issue Overview
Looking for a way to let a Pods field assume the value of another field in the WordPress database (Pods field or non-Pods field) through defining fields in the Pods GUI.

## Possible Solution
How I would love to see it work:
When defining a field in the Pods GUI, I'd simply set the 'Field Type' to 'database field', select the field, and boom, it simply takes that field value. A selection field 'sync fields permanently' allows this to be a once-off sync or always sync the values of these fields. A dropdown box would additionally gives these choices: 'sync both ways', 'slave' or 'master'.

Since Pods allows to define fields, and recognizes many field types, I think it's logical to let Pods define this type of field too.

## Related Issues and/or PRs
I checked with @jimtrue: the above solution currently does not exist in the Pods GUI. The alternative method of using the pre_save filter or a post_save action hook (https://pods.io/docs/code/filter-reference/pods_api_pre_save_pod_item_podname) therefore does not offer a solution.

**Comment by quasel** [comment]
What's the use case for that?  Don't think that's a priority - sorry ;) especially for "sync" sound more like a thing maybe solved with a relationship or "default" values?

**Comment by coercoer** [comment]
Use case for synchronizing database fields? I assume I don't have to explain that people want to let one database field assume the value of another. So perhaps I misunderstand your question?
Calculating fields on the basis of another field is already an accepted change request for Pods that you are working on, whilst this request is much more basic; one field = another field. Perhaps with calculation this could simply be merged as a request because it would be the simplest of calculations: Field A = Field B + 0. But that would potentially limit the synced fields only to mathematical fields. Whilst what I would like is that any similar field type can be synchronized.

Perhaps you need examples?
Here is one:
I have some fields that were added through plugins, such as a profit calculation. The problem is that the plugin does a great job in reporting but it does not take some cost into account and I made my own extended WooCommerce fields ('Product CPT extended through Pods) to add certain cost elements. I then want to synchronize the Plugin field with my own calculated profit per transaction of WooCommerce. That way I control the profit calculation method, but I can still use the plugin's reporting and presentation layer in the backend of WordPress. 

That's just one case.
